### PR TITLE
Update Brick-Hill to 1.0.3

### PR DESCRIPTION
**Pull request by okdargy**
I made this before but I totally forgot about it so I'm making another one :) 

Previous One: https://github.com/PreMiD/Presences/pull/2793

- Redid how it looks on your profile (Viewing Page: | Dashboard)
- Added preciese info (game name, clan name, user name)
- Added blog & API presence
- Added promocodes (secret/new page)

image
image
image

### CODE REVIEW

**Review by rxri — CHANGES_REQUESTED** [review]
Fix codefactor and move all `presenceData.startTimestamp = browsingStamp` lines to just `presenceData` object

File: websites/B/Brick Hill/presence.ts [hunk 1]
```diff
@@ -2,65 +2,123 @@ const presence = new Presence({
...
+
+  // API Presence
+  if (document.location.origin.includes("api.brick-hill.com")) {
+    if (document.location.pathname.includes("/docs")) {
+      presenceData.details = "Viewing Page:";
+      presenceData.state = "API Docs";
+    }
   }
 
   if (presenceData.details == null) {
```
Comment (SlowLife1661) [inline websites/B/Brick Hill/presence.ts:118]:
```suggestion
  if (!presenceData.details) {
```

File: websites/B/Brick Hill/presence.ts [hunk 2]
```diff
@@ -2,65 +2,123 @@ const presence = new Presence({
...
   browsingStamp = Math.floor(Date.now() / 1000);
-
 presence.on("UpdateData", async () => {
-  const presenceData: PresenceData = {
-    largeImageKey: "brick-hill"
+  const presenceData = {
+    largeImageKey: "brick-hill",
+    details: "Viewing Page:",
+    state: "404",
+    startTimestamp: Math.floor(Date.now() / 1000)
```
Comment (SlowLife1661) [inline websites/B/Brick Hill/presence.ts:10]:
```suggestion
    startTimestamp: browsingStamp
```

**Review by Bas950 — DISMISSED** [review]

File: websites/B/Brick Hill/presence.ts [hunk 3]
```diff
@@ -2,65 +2,123 @@ const presence = new Presence({
...
+  }
+
+  if (presenceData.details) {
     presence.setTrayTitle();
     presence.setActivity();
-  } else {
+  } else 
     presence.setActivity(presenceData);
-  }
+
```
Comment (ririxidev) [inline websites/B/Brick Hill/presence.ts:123]:
```suggestion
```

File: websites/B/Brick Hill/presence.ts [hunk 4]
```diff
@@ -2,65 +2,123 @@ const presence = new Presence({
     clientId: "797879854343127040"
   }),
   browsingStamp = Math.floor(Date.now() / 1000);
-
```
Comment (ririxidev) [inline websites/B/Brick Hill/presence.ts:5]: add blank line here?

**Review by rxri — CHANGES_REQUESTED** [review]

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**